packages/cadl-ranch-specs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure-tools/cadl-ranch-specs",
-  "version": "0.16.2",
+  "version": "0.16.3",
```

Comment:
I am not sure why specs is patch update (0.16.2 to 0.16.3), while cadl-ranch-expect and cadl-ranch is minor update?

Could you have Chenjie for double check?

Reply:
With reference of last time, I find all version is bumped to `0.X+1.0`. So I think 0.17.0 is better. add @tadelesh to confirm

Reply:
0.17.0 is better for tsp upgrade.

Reply:
done